LiskHQ/lisk-hub
# Implement logout button - Closes #540

### What was the problem?
No Logout button
### How did I fix it?
Implemented logout button
### How to test it?
- Login to your page and click logout it should open dialog with logout confirmation
- When less than 1 min opens dashboard `Timeout in mm:ss`
- When timer reaches 0:00 opens `Session timeout` dialog
All designs are accessible via this link
https://taikonauten.atlassian.net/browse/L20-411
### Review checklist
- The PR solves #540
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
